### PR TITLE
CNF 792 RN: Performance Profile Creator tool

### PR DESCRIPTION
**Pull request by kquinn1204**
New feature Release Notes description for Performance Profile Creator.
Link to preview https://deploy-preview-30935--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-performance-creator-tool
Issue: https://issues.redhat.com/browse/CNF-792

Affects release notes for 4.8

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -97,6 +97,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-scale"]
 === Scale
 
+[id="ocp-4-8-performance-creator-tool"]
+==== Creating a Performance Profile
```
Comment (StephenJamesSmith) [inline release_notes/ocp-4-8-release-notes.adoc:101]: performance profile

**Comment by kquinn1204** [comment]
release notes posted to here https://deploy-preview-30935--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-scale

**Comment by kquinn1204** [comment]
Ok got message in email from Swati 
"Hi Kevin,
Good idea to keep it basic. This looks good to me!

Thanks for taking care of this,
Regards"

**Comment by sagidlow** [comment]
@kquinn1204 Please squash your commits into 1 commit and remove the WIP label.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -242,6 +242,11 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-scale"]
 === Scale
 
+[id="ocp-4-8-performance-creator-tool"]
+==== Creating a performance profile
+You can now create a performance profile using the Performance Profile Creator (PPC) tool. The tool consumes `must-gather` data from the cluster, several user-supplied profile arguments, and generates a performance profile that is appropriate for your hardware and topology.
```
Comment (sagidlow) [inline release_notes/ocp-4-8-release-notes.adoc:247]: This sentence: 

> The tool consumes `must-gather` data from the cluster, several user-supplied profile arguments, and generates a performance profile that is appropriate for your hardware and topology.

It treats the three items in the sentence as if they are all the kind of list item in a series. However, it seems like the actual tool is gathering data and arguments, and generating a performance profile. I would maybe reword it a little bit?

**Review by sagidlow — COMMENTED** [review]
@kquinn1204 -- I had 1 sort of nit comment to address before merge. 
Please also squash your commits into 1 and remove the WIP label. 
Also, please update your initial comment with the preview link and any associated BZ/JIRA links. 

> The docs manual describes the [correct structure](https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#bookmark=id.1cwpsjpyzs43) for the title and first comment of your PRs. 
> Each PR title must be structured like so (unless there is no underlying issue ID that your PR addresses):
> [GH#<Issue ID>][BZ<BZ ID>][<Jira ID>]: <Short description of your PR>
> The first comment should include a direct link to the preview, the branches that this PR affects, and a direct link to the underlying issue (if your PR addresses one). You may add other meta information if needed.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (kquinn1204) [reply release_notes/ocp-4-8-release-notes.adoc:247]: HI @sagidlow was going to adjust to this 
The tool consumes `must-gather` data from the cluster, several user-supplied profile arguments, and using this information generates a performance profile that is appropriate for your hardware and topology.

The tool itself does not gather the data it uses the data captured in must-gather with various command-line arguments to generate a profile.

Comment (sagidlow) [reply release_notes/ocp-4-8-release-notes.adoc:247]: I would suggest rewording to something like this:

> The tool consumes `must-gather` data from the cluster and several user-supplied profiles arguments, and using this information it generates a performance profile that is appropriate for your hardware and topology. The tool itself does not gather the data. It uses the data captured in `must-gather` with various command line arguments to generate a profile.

Comment (kquinn1204) [reply release_notes/ocp-4-8-release-notes.adoc:247]: sorry was not clear I think to keep it simple I will adjust to this "The tool consumes must-gather data from the cluster, several user-supplied profile arguments, and using this information generates a performance profile that is appropriate for your hardware and topology." I am trying to keep RN information quite minimal. They can read more in the link.

Comment (sagidlow) [reply release_notes/ocp-4-8-release-notes.adoc:247]: My apologies I thought the "tool itself" portion was part of the text you wanted to add to the RN. The suggestion below for the first sentence. There are still some minor grammar issues for the initial list so if you could please address that. 

> The tool consumes `must-gather` data from the cluster and several user-supplied profiles arguments, and using this information it generates a performance profile that is appropriate for your hardware and topology.

**Comment by kquinn1204** [comment]
@sagidlow all sorted hopefully. Sorry for all the to and fro and thanks for your patience.
